Makefile

```diff
@@ -21,9 +21,9 @@ release/sgp30:
 	cp -rl embedded-common/* "$${pkgdir}" && \
```

Comment:
These cp commands also needs to be adjusted.

Maybe also mention this adjustment in the commit message.

Reply:
Indeed. I'll blame it on the lunch break interrupting the workflow..